capstone/src/main/java/com/google/sps/servlets/StudySetServlet.java

```diff
@@ -42,13 +42,15 @@
 public class StudySetServlet extends HttpServlet {
   private final String search_sql_statement = 
     "SELECT COUNT(card.study_set_id), study_set.id, study_set.title, study_set.description, " +
-    "study_set.subject,  university.name, user_info.user_name FROM study_set JOIN university " +
+    "study_set.subject, university.name, user_info.user_name FROM study_set JOIN university " +
     "ON university.id = study_set.university_id JOIN user_info ON study_set.owner_id = user_info.id " +
-    "JOIN card ON card.study_set_id = study_set.id GROUP BY study_set.id, university.id, user_info.id";
+    "JOIN card ON card.study_set_id = study_set.id WHERE card.front ILIKE  ? OR card.back ILIKE ? OR " +
```

Comment:
extra space?

Reply:
the extra space between ILIKE and ?  ...

Reply:
Right.

Reply:
Fixed the space